.github/workflows/code_review.yml

```diff
@@ -12,9 +12,10 @@
       - name: Checkout repository
         uses: actions/checkout@v3
       - name: Code Review
-        uses: freeedcom/ai-codereviewer@main
+        uses: ./
         with:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
-          OPENAI_API_MODEL: "gpt-4-1106-preview"
+          OPENAI_API_MODEL: "gpt-4o-mini"
```

Comment:
The model `gpt-4o-mini` may not be the intended model for your use case. Ensure that this model meets your requirements, as it might have limitations compared to the original `gpt-4-1106-preview`.